tests/integration/test_ingressed_tls.py

```diff
@@ -1,180 +1,177 @@
-    )
-
-    await asyncio.gather(
-        ops_test.model.wait_for_idle(
-            apps=[APP_NAME, SSC_APP_NAME, TRAEFIK_APP_NAME],
-            status="active",
-            raise_on_blocked=True,
-            timeout=10000,
-            raise_on_error=False,
-        ),
-    )
-
-
-@pytest.mark.setup
-@pytest.mark.abort_on_fail
-async def test_push_tracegen_script_and_deps(ops_test: OpsTest):
-    await ops_test.juju("scp", TRACEGEN_SCRIPT_PATH, f"{APP_NAME}/0:tracegen.py")
-    await ops_test.juju(
-        "ssh",
-        f"{APP_NAME}/0",
-        "python3 -m pip install opentelemetry-exporter-otlp-proto-grpc opentelemetry-exporter-otlp-proto-http",
-    )
-
-
-@pytest.mark.setup
-@pytest.mark.abort_on_fail
-async def test_relate(ops_test: OpsTest):
-    await ops_test.model.integrate(APP_NAME + ":certificates", SSC_APP_NAME + ":certificates")
-    await ops_test.model.integrate(
-        SSC_APP_NAME + ":certificates", TRAEFIK_APP_NAME + ":certificates"
-    )
-    await ops_test.model.integrate(APP_NAME + ":ingress", TRAEFIK_APP_NAME + ":traefik-route")
-    await ops_test.model.wait_for_idle(
-        apps=[APP_NAME, SSC_APP_NAME, TRAEFIK_APP_NAME],
-        status="active",
-        timeout=1000,
-    )
-
-
-@pytest.mark.abort_on_fail
-async def test_verify_ingressed_trace_http_upgrades_to_tls(ops_test: OpsTest, nonce):
-    tempo_host = await get_tempo_host(ops_test)
-    # IF tempo is related to SSC
-    # WHEN we emit an http trace, **unsecured**
-    await emit_trace(
-        f"http://{tempo_host}:4318/v1/traces", nonce=nonce, ops_test=ops_test
-    )  # this should fail
-    # THEN we can verify it's not been ingested
-    assert get_traces_patiently(tempo_host, nonce=nonce)
-
-
-@pytest.mark.abort_on_fail
-async def test_verify_ingressed_trace_http_tls(ops_test: OpsTest, nonce, server_cert):
-    tempo_host = await get_tempo_host(ops_test)
-    await emit_trace(
-        f"https://{tempo_host}:4318/v1/traces", nonce=nonce, ops_test=ops_test, use_cert=True
-    )
-    # THEN we can verify it's been ingested
-    assert get_traces_patiently(tempo_host, nonce=nonce)
-
-
-@pytest.mark.abort_on_fail
-async def test_verify_ingressed_traces_grpc_tls(ops_test: OpsTest, nonce, server_cert):
-    tempo_host = await get_tempo_host(ops_test)
-    await emit_trace(
-        f"{tempo_host}:4317", nonce=nonce, proto="grpc", ops_test=ops_test, use_cert=True
-    )
-    # THEN we can verify it's been ingested
-    assert get_traces_patiently(tempo_host, nonce=nonce)
-
-
-@pytest.mark.teardown
-@pytest.mark.abort_on_fail
-async def test_remove_relation(ops_test: OpsTest):
-    await ops_test.juju(
-        "remove-relation", APP_NAME + ":certificates", SSC_APP_NAME + ":certificates"
-    )
-    await asyncio.gather(
-        ops_test.model.wait_for_idle(
-            apps=[APP_NAME], status="active", raise_on_blocked=True, timeout=1000
-        ),
-    )
+# import asyncio
```

Comment:
why are we commenting out a whole module? 
If it's no longer relevant, let's remove it. 
If there's a really good reason to keep it around, write that in a FIXME/TODO.

Reply:
how was this issue resolved exactly?

Reply:
Added a TODO